server/command_hooks.go

```diff
@@ -44,44 +57,45 @@ func (p *Plugin) emitStatusChange() {
 func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
 	configuration := p.getConfiguration()
 
-	if !strings.HasPrefix(args.Command, "/"+CommandTrigger) {
-		return &model.CommandResponse{
-			ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
-			Text:         fmt.Sprintf("Unknown command: " + args.Command),
-		}, nil
-	}
+	if strings.HasPrefix(args.Command, "/"+CommandTrigger) {
+
```

Comment:
Nit: remove this line

Reply:
This line is needed though to check between the `/demo_plugin` and `/crash` commands...   Am I misunderstanding what you are asking?

Reply:
I was talking about the empty line. But never mind, we can fix this later.